src/gc/impl/conservative/gc.d

```diff
@@ -163,7 +167,10 @@ class ConservativeGC : GC
 
     this()
     {
+        import core.exception : onConfigurationError;
         //config is assumed to have already been initialized
+        if (config.fork && config.parallel > 0)
+            onConfigurationError("Can't mix threads and fork strategies for GC marking");
```

Comment:
Is this actually an existing limitation? I can think of use cases where a limited number of threads in the marking process can be helpful.

Reply:
I am not convinced, but we will let the user profile and decide. https://github.com/dlang/druntime/pull/2604/commits/c734c8ef8f5f311f893111ace6a2523f099ea403  allows mixed forked  + threaded marking